src/cli/Create.cpp

```diff
@@ -132,7 +132,7 @@ QSharedPointer<PasswordKey> Create::getPasswordFromStdin()
     out.flush();
     QString password = Utils::getPassword();
 
-    if (password.length() > 0) {
+    if (password.isEmpty()) {
```

Comment:
@droidmonkey shouldn't that be `!password.isEmpty()`??

Reply:
yes sir! I need to fix tests too